### DISCUSSION TITLE
Is there an example for ios and wasm?

### DISCUSSION THREAD

**Discussion by CeSun** [dotnet/Silk.NET#865] [Q&A] [answered]
The code repository seems to only have examples for pc and android.

**Comment by Perksey** [comment, marked as answer]
iOS is not officially supported (it never graduated from "experimental" status because there were no community contributors willing to help us get it working on an actual device). So while the code is there, it might not work and because of that we don't want to provide examples.

WASM is entirely unsupported.

Hope this helps.

**Reply by CeSun** [reply]
Thank you for the answer and your library, it really works well.
